src/main/java/cs2103/v15_1j/jimjim/command/SaveLocationCommand.java

```diff
@@ -23,20 +23,28 @@ public String getSavePath() {
     @Override
     public UIFeedback execute(ControllerStates conStates) {
     	String backup = conStates.config.savePath;
+    	File file = null;
     	
-    	try {
-    		File f = new File(backup);	
-    	} catch (Exception e) {
+    	if (savePath.contains(" ")) {
```

Comment:
It creates two folders. The space is allowed only if you use a backslash.

On Tuesday, 5 April 2016, Tran Tien Dat <notifications@github.com> wrote:

> In src/main/java/cs2103/v15_1j/jimjim/command/SaveLocationCommand.java
> <https://github.com/cs2103jan2016-v15-1j/main/pull/226#discussion_r58491235>
> :
>
> >      	
> > -    	try {
> > -    		File f = new File(backup);	
> > -    	} catch (Exception e) {
> > +    	if (savePath.contains(" ")) {
>
> Why is this invalid? We can have space in file / folder name what?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/cs2103jan2016-v15-1j/main/pull/226/files/89a6d808151dae517bec90547ee9f6a0c28f8498..de18f8254985571e283763a90a40b4951983eb02#r58491235>
>


-- 

Shaun Thium (Mr.)
*National University of Singapore*
B.Comp (Computer Science) | University Scholar's Programme